app/views/root/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<div id="root">
+<h3> Hello </h3>
```

Comment:
- [ ] You shouldn't include any other lines here, the only line that should be here is `<div id="root">`, kindly remove the other line.